package.json

```diff
@@ -12,7 +12,7 @@
     "build": "cross-env NODE_ENV=production babel src --out-dir dist",
     "lint": "eslint --format=node_modules/eslint-formatter-pretty src test *.js",
     "preversion": "npm run build",
-    "spec": "npm run build && jest test/*.spec.js",
+    "spec": "npm run build && jest test/format.spec.js",
```

Comment:
Let's keep this to `*.spec.js`. There's probably going to be more tests added here in the future